kyuubi-server/src/main/scala/org/apache/kyuubi/sql/trino/TrinoGetSchemas.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.sql.trino
+
+import org.apache.kyuubi.sql.plan.KyuubiTreeNode
+
+/////////////////////////////////////////////////////////////////////////////////////////
+// This file contains all Trino JDBC operation nodes which are parsed from statement
```

Comment:
After that, there are GetTable/GetType and other Nodes that need to be parsed, and here the `TrinoGetSchemas.scala` naming is not appropriate